scripts/setup-database.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+distrofull=$(cat /etc/issue);
+distro=${distrofull:: -6};
+echo "Enter number of the Database Client you prefer:";
```

Comment:
See this answer, it looks tidy https://askubuntu.com/a/1716/740820
Shall we try in that manner?